src/common/test-webserver.c

```diff
@@ -34,22 +34,22 @@ typedef struct {
   CockpitWebServer *web_server;
   gchar *localport;
   gchar *hostport;
-} TestCase;
+} TestFixture;
 
 typedef struct {
   const gchar *cert_file;
   gboolean local_only;
   gboolean inet_only;
   CockpitWebServerFlags server_flags;
-} TestFixture;
+} TestCase;
```

Comment:
Hah, thanks. This affects pretty much every unit test, so one of these days you could perhaps apply that seddery to the others. :grinning: